src/nixos-remote.sh

```diff
@@ -100,15 +100,17 @@ timeout_ssh_() {
 ssh_() {
   ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no "$ssh_connection" "$@"
 }
-nixCopy() {
-  NIX_SSHOPTS='-o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no' nix copy --extra-experimental-features nix-command "$@"
+nix_copy() {
+  NIX_SSHOPTS='-o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no' nix copy \
+    --extra-experimental-features 'nix-command flakes' \
+    "$@"
 }
 nix_build() {
-  nix \
-    --experimental-features flakes build \
-    --extra-experimental-features nix-command \
+  nix build \
```

Comment:
Potentially we want to more logs here in case something goes wrong with the build:

```suggestion
  nix build -L \
```

Reply:
I did notice nixos-remote output is either really loud and useless or super silent and I don't actually know if anything is happening. Maybe printing nix build logs should be a flag?

Reply:
We can also leave this for another PR...

Reply:
sure, just merge this how it is and I will make a follow-up PR soon.